style/sass/README.md

```diff
@@ -27,29 +27,178 @@
 
-## Order
-
-* Use alphabetical order for declarations.
-* Place `@extends` and `@includes` at the top of your declaration list.
-* Place media queries directly after the declaration list.
-* Place pseudo-classes and pseudo-elements second.
-* Place nested elements third.
-* Place nested classes fourth.
-
 ## Selectors
 
-* Don't use ID's for style.
-* Avoid over-qualified selectors: `h1.page-title`, `div > .page-title`
 * Use meaningful names: `$visual-grid-color` not `$color` or `$vslgrd-clr`.
-* Be consistent about naming conventions for classes. For instance, if a project is using BEM, continue using it, and if it's not, do not introduce it.
 * Use ID and class names that are as short as possible but as long as necessary.
 * Avoid nesting more than 3 selectors deep.
 * Avoid using comma delimited selectors.
 * Avoid nesting within a media query.
-* Don't concatenate selectors using Sass's parent selector (`&`).
 
 ## Organization
 
 * Use a `base` directory for styling element selectors, global variables, global extends and global mixins.
 * Use HTML structure for ordering of selectors. Don't just put styles at the bottom of the Sass file.
 * Avoid having files longer than 100 lines.
+
+---
+
+## Selectors
+
+### Selector specificity
+
+- Don't use ID selectors.
+- Avoid over-qualified selectors, e.g. `h1.page-title`.
+
+<details>
+
+#### Code examples
+
+`h1.page-title` carries a specificity of 2, but can be reduced to 1 by removing
+the `h1` type selector:
+
+```diff
+-h1.page-title
++.page-title {
+   // …
+ }
+```
+
+#### Motivation
+
+Using an ID in a selector increases its specificity, making it more difficult
+to work with alongside class selectors. Furthermore, because IDs must be unique
+within an HTML document, using them as CSS selectors limits reusability.
+
+#### Resources
+
+- Learn about [how specificity is calculated][specificity-calculation].
+
+[specificity-calculation]: https://www.w3.org/TR/selectors-3/#specificity
+
+</details>
+
+### Selector naming
+
+- Use lowercase characters and hyphens (sometimes referred to as hyphen-case,
+  dash-case, or kebab-case) when naming selectors.
+- Be consistent about naming conventions. For instance, if a project is using
+  BEM, continue using it, and if it's not, don't introduce it.
+- Don't uses Sass parent selectors (`&`) to concatenate selector names.
+
+<details>
+
+#### Code examples
+
+Use lowercase characters and hyphens in selector names:
+
+```scss
+.class-name {
+  // …
+}
+```
+
+Don't concatenate selector names:
+
+```scss
+.class {
+  &__child-class {
+    // …
+  }
+}
+```
+
+#### Motivation
+
+Concatenating selector names makes it more difficult to search and find
+selectors in the codebase.
+
+</details>
+
+## General syntax and formatting
+
+### Declarations block ordering
+
+- Order declarations alphabetically.
+- Order items within the declaration block in the following order:
+    1. Sass at-rules, e.g. `@include`
+    1. CSS properties
```

Comment:
should these numbers be 1, 2, 3, 4... instead of all 1s?

Reply:
Fun little Markdown tip: these get rendered as a proper ordered list, 1, 2, 3, 4.

Raw Markdown:

```
1. item
1. item
1. item
```

Rendered Markdown:

1. item
1. item
1. item

I like to use all `1`s in the Markdown, because it makes shuffling or moving anything in the list much easier, as you don't need to then go through and re-order the numbers.

Reply:
But isn't the point of Markdown to create a human readable text file too?

Reply:
Perhaps…

It is [billed as](https://daringfireball.net/projects/markdown/) an “…easy-to-read, easy-to-write plain text format.”

Though, I feel like the consumption of this content happens more often in the rendered state. In HTML, each item within an ordered list are all represented as `<li>`, which is comparable to using all `1`s in Markdown.

Reply:
Casting a vote for using 1s. We use it elsewhere, and it's a pretty common pattern for a lot of Markdown documentation.

Reply:
TIL! thanks for explaining :)

Reply:
I’m going to sticky with all `1`s.